test/result_test.rb

```diff
@@ -614,6 +614,47 @@ class ResultTest < TinyTds::TestCase
 
       if sqlserver?
 
+        describe 'using :message_handler' do
+          messages = []
```

Comment:
Could this be a `let(:messages) { Array.new }`

Reply:
Done